### PR TITLE
Refactor/packages

### PR DESCRIPTION
**Pull request by egolesor**

### CODE REVIEW

File: battleship-service/src/main/java/dreamteam/battleship/service/springcontroller/preparation/PlayerOrganizer.java [hunk 1]
```diff
@@ -47,7 +49,7 @@ public PlayerOrganizerResponse preparePlayer(HttpSession session) {
             }
         }
         logger.debug(END);
-        return new PlayerOrganizerResponse(gameController.isReadyToPlay());
+        return new dreamteam.battleship.service.springcontroller.model.response.PlayerOrganizer(gameController.isReadyToPlay());
```
Comment (danone771909) [inline battleship-service/src/main/java/dreamteam/battleship/service/springcontroller/preparation/PlayerOrganizer.java]: import proper package

File: battleship-service/src/main/java/dreamteam/battleship/service/springcontroller/registration/Registration.java [hunk 2]
```diff
@@ -29,15 +26,15 @@
     protected HttpSession session;
 
     @RequestMapping(method = RequestMethod.GET, path = "/register")
-    public RegistrationResponse register(@RequestParam(name = "name") String name,
-                                         @RequestParam(name = "surname") String surname) {
+    public dreamteam.battleship.service.springcontroller.model.response.Registration register(@RequestParam(name = "name") String name,
```
Comment (danone771909) [inline battleship-service/src/main/java/dreamteam/battleship/service/springcontroller/registration/Registration.java]: import proper package

File: battleship-service/src/test/java/dreamteam/battleship/service/springcontroller/preparation/PreparePlayerTest.java [hunk 3]
```diff
@@ -18,7 +20,7 @@
 
     @Test
     public void testIfGameControllerDoesntLetTheGameStartWIthOnePlayer(){
-        PlayerOrganizer organizer = new PlayerOrganizer();
+        dreamteam.battleship.service.springcontroller.preparation.PlayerOrganizer organizer = new dreamteam.battleship.service.springcontroller.preparation.PlayerOrganizer();
```
Comment (danone771909) [inline battleship-service/src/test/java/dreamteam/battleship/service/springcontroller/preparation/PreparePlayerTest.java]: import proper package

File: battleship-service/src/test/java/dreamteam/battleship/service/springcontroller/registration/RegistrationTest.java [hunk 4]
```diff
@@ -42,9 +41,9 @@ public void testIfGeneratedKeyAreUnique(boolean isUnique){
     public void testIfTheGeneratedKeyReturnsAfterRegistering(){
         MockHttpSession session = new MockHttpSession();
 
-        Registration registration = new Registration();
+        dreamteam.battleship.service.springcontroller.registration.Registration registration = new dreamteam.battleship.service.springcontroller.registration.Registration();
```
Comment (danone771909) [inline battleship-service/src/test/java/dreamteam/battleship/service/springcontroller/registration/RegistrationTest.java]: import proper package

File: battleship-service/src/main/java/dreamteam/battleship/service/springcontroller/model/response/Register.java [hunk 5]
```diff
@@ -0,0 +1,15 @@
+package dreamteam.battleship.service.springcontroller.model.response;
+
+import dreamteam.battleship.service.springcontroller.model.Player;
+
+/**
+ * Created by ehsan on 18.07.16.
```
Comment (LIttleAncientForestKami) [inline battleship-service/src/main/java/dreamteam/battleship/service/springcontroller/model/response/Register.java]: When you do Javadocs on class, capitalize your name, and beside who created the class and when, add actual Javadoc.

File: battleship-service/src/main/java/dreamteam/battleship/service/springcontroller/model/response/Restart.java [hunk 6]
```diff
@@ -0,0 +1,7 @@
+package dreamteam.battleship.service.springcontroller.model.response;
+
+/**
+ * The class is to return the information about the restart
```
Comment (LIttleAncientForestKami) [inline battleship-service/src/main/java/dreamteam/battleship/service/springcontroller/model/response/Restart.java]: The class is redundant on class Javadoc. Everybody knows it's a class. Remove please. Also, class does not return.

File: battleship-service/src/main/java/dreamteam/battleship/service/springcontroller/play/Playing.java [hunk 7]
```diff
@@ -54,15 +56,15 @@ public ShootResponse shoot(@RequestParam(name = "fieldNumber") int fieldNumber)
     }
 
     @RequestMapping(method = RequestMethod.GET, path = "/turnstatus")
-    public TurnStatusResponse turnStatus(HttpSession session){
+    public TurnStatus turnStatus(HttpSession session){
         // this method will call iterally, so i dont think that logging is a good idea
```
Comment (LIttleAncientForestKami) [inline battleship-service/src/main/java/dreamteam/battleship/service/springcontroller/play/Playing.java]: * When I means myself it should ALWAYS be capitalized. 
* iterally? Either iteratively or literally, I think?
* also, if this is a method comment, why not Javadoc?

File: battleship-service/src/main/java/dreamteam/battleship/service/springcontroller/play/Playing.java [hunk 8]
```diff
@@ -71,9 +73,9 @@ private ShootResponse winnerResponse() {
      * @param fieldNumber
      * @return
      */
-    private ShootResponse handleShoot(HttpSession session, int fieldNumber) {
+    private Shoot handleShoot(HttpSession session, int fieldNumber) {
         logger.debug("Handling the shoot");
```
Comment (LIttleAncientForestKami) [inline battleship-service/src/main/java/dreamteam/battleship/service/springcontroller/play/Playing.java]: the shot or the shooting

File: battleship-service/src/main/java/dreamteam/battleship/service/springcontroller/model/response/Placing.java [hunk 9]
```diff
@@ -8,13 +8,13 @@
 /**
  * Created by egolesor on 18.07.16.
  */
-public class PlacingResponse {
+public class Placing {
```
Comment (LIttleAncientForestKami) [inline battleship-service/src/main/java/dreamteam/battleship/service/springcontroller/model/response/Placing.java]: `Placing` is a verb, you know? So is `Shoot`. Are you certain you wanted your classes named like that?

**Pull request merged**